### PR TITLE
Used model's Options.label/label_lower where applicable.

### PR DESCRIPTION
**Pull request by felixxm**
We can reuse [model's `Options` properties](https://github.com/django/django/blob/cf493e5c819f5ee49b96954f026bec722e19d9c3/django/db/models/options.py#L132-L138).

### CODE REVIEW

File: django/contrib/admin/checks.py [hunk 1]
```diff
@@ -607,8 +607,8 @@ def _check_readonly_fields_item(self, obj, field_name, label):
             except FieldDoesNotExist:
                 return [
                     checks.Error(
-                        "The value of '%s' is not a callable, an attribute of '%s', or an attribute of '%s.%s'." % (
```
Comment (felixxm) [inline django/contrib/admin/checks.py]: I didn't make any cosmetic edits (as wrapping at 79 chars) to keep diff as small as possible.

Comment (carltongibson) [reply django/contrib/admin/checks.py]: We're touching this line. I'd be inclined to fix it...

Comment (felixxm) [reply django/contrib/admin/checks.py]: OK, wrapped.

**Comment by felixxm** [comment]
@carltongibson @hramezani  Thanks for reviews :+1:

**Pull request merged**